bundles/org.openhab.binding.millheat/src/main/java/org/openhab/binding/millheat/internal/model/ModeType.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.millheat.internal.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * The {@link ModeType} represents a type of mode the user can set in the app.
+ *
+ * @author Arne Seime - Initial contribution
+ */
+public enum ModeType {
+    AlwaysHome(-1),
```

Comment:
Enums should also follow the Java code conventions and be like constants. So all caps separated by _, if they are used as Gson models you could use `@SerializedName`